x-pack/plugins/snapshot_restore/__jest__/client_integration/home.test.ts

```diff
@@ -32,16 +32,6 @@ jest.mock('@kbn/i18n-react', () => {
   };
 });
 
-jest.mock('../../common/constants', () => {
-  const original = jest.requireActual('../../common/constants');
-
-  return {
-    ...original,
-    // Mocking this value to a lower number in order to more easily trigger the max snapshots warning in the tests
-    SNAPSHOT_LIST_MAX_SIZE: 2,
-  };
-});
-
```

Comment:
This old mock was used for testing a warning callout about only showing a limited number of snapshots. The warning was removed in [#110266](https://github.com/elastic/kibana/pull/110266)